packages/bitcore-client/bin/wallet-bump-tx

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('../ts_build/program');
+const { Wallet } = require('../ts_build/wallet');
+const promptly = require('promptly');
+const ethers = require('ethers');
+
+program
+  .version(require('../package.json').version)
+  .option('--name <name>', 'REQUIRED - Wallet Name')
+  .option('--txid [txid]', 'REQUIRED (or provide raw tx) - string transaction id')
+  .option('--rawtx [rawtx]', 'REQUIRED (or provide tx id) - string raw transaction')
+  .option(
+    '--gwei [gwei]',
+    'optional  (defaults to current market rate) for use with EVM based chains - number gwei (in eth/matic)'
+  )
+  .option('--path [path]', 'optional - Custom wallet storage path')
+  .option('--storageType [storageType]', 'optional - name of the database to use (default Level)')
+  .parse(process.argv);
+
+const main = async () => {
+  const { name, path, storageType, txid, rawtx, gwei } = program;
+  let wallet;
+
+  try {
+    wallet = await Wallet.loadWallet({ name, path, storageType });
+
+    let existingTx;
+    if (txid) {
+      existingTx = JSON.parse(await wallet.getTransactionByTxid({ txid }));
+    } else if (rawtx) {
+      existingTx = ethers.utils.parseTransaction(rawtx);
+    } else {
+      throw new Error('Must provide either a raw transaction or transaction id.');
+    }
+
+    const existingGasPriceInWei = Number(existingTx.gasPrice);
+    let newFeeInWei;
+    if (gwei) {
+      newFeeInWei = Number(gwei) * 1000000000;
```

Comment:
I would suggest making 1000000000 a variable and using it everywhere we have `1000000000`. Things like this make it easy to miss a zero in one place or something